--- a/apps/dust_cli/mix.exs	2026-05-25 11:05:40.632083802 -0500
+++ b/apps/dust_cli/mix.exs	2026-05-25 11:07:33.614267129 -0500
@@ -26,7 +26,6 @@
   defp deps do
     [
       {:jason, "~> 1.4"},
-      {:burrito, "~> 1.0"},
       {:owl, "~> 0.12"},
       {:websockex, "~> 0.4.3"}
     ]
@@ -35,16 +34,8 @@
   defp releases do
     [
       dustctl: [
-        steps: [:assemble, &Burrito.wrap/1],
-        burrito: [
-          targets: [
-            linux_x86_64:   [os: :linux,   cpu: :x86_64],
-            linux_aarch64:  [os: :linux,   cpu: :aarch64],
-            macos_x86_64:   [os: :darwin,  cpu: :x86_64],
-            macos_aarch64:  [os: :darwin,  cpu: :aarch64],
-            windows_x86_64: [os: :windows, cpu: :x86_64]
-          ]
-        ]
+        steps: [:assemble],
+        applications: [dust_cli: :permanent]
       ]
     ]
   end
